virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -27,6 +27,8 @@ In {VirtProductName} clusters installed using installer-provisioned infrastructu
 
 * To use proxy with {VirtProductName}, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager].
 
+* Before you migrate virtual machines to a cluster, ensure that all nodes in a single pool use the same type of CPU. For example, place nodes that use AMD processors in one pool and place nodes that use Intel processors in a different pool. Do not mix different CPU types in the same pool.
```

Comment:
/hold

I find this text confusing. What is to "migrate virtual machines to a cluster"? You can either import a VM to cluster, or live-migrate a VM *in* a cluster. I suppose this release note refers to the latter, but typical users do not choose to migrate, they can do nothing "before" migrations. Also, I don't know what is the "pool" this text refers to.

Reply:
OK, thanks @dankenigsberg.  I can rewrite this with the new information supplied by Israel re: labelselectors and affinity.

Reply:
Great. How can we make sure to use only well-defined  concepts (unlike "cpu pools")?

Reply:
A few things I can think of:

Reply:
Thanks for all the help :)

Reply:
No problem @kobi86. It helps me to learn about these concepts (labelselectors, etc) so there is good in that :-)

Reply:
@kobi86 @bgaydosrh can you drop this "cpu pool" concept from the BZ and PR?